test/test_quantizer.py

```diff
@@ -222,3 +225,110 @@ def forward(self, x):
         self.assertTrue('z' in eagerDict and 'z.1' in activationDict)
         self.assertAlmostEqual(eagerDict["z"][0], activationDict["z.1"][0], places=15)
         self.assertAlmostEqual(eagerDict["z"][1], activationDict["z.1"][1], places=15)
+
+    def test_compare_qparam_eager_script_default(self):
```

Comment:
Nit: what do 'qparam' and 'default' words mean in the name of this test?

Reply:
I plan to rename this to test_default in later PR..
default just means simple/naive situation, maybe I can rename that to test_simple